From 27cfdc75084e57f22ba93c54f2387a7c47378d90 Mon Sep 17 00:00:00 2001
From: Felix Geyer <debfx-kde@fobos.de>
Date: Tue, 8 Nov 2011 20:44:05 +0100
Subject: [PATCH 19/20] Remove unused "Remember authorization" checkboxes.

The checkboxes were visible in the UI but the feature
isn't actually implemented.
---
 AuthDialog.ui |   33 ---------------------------------
 1 files changed, 0 insertions(+), 33 deletions(-)

diff --git a/AuthDialog.ui b/AuthDialog.ui
index 27480ff..3b9bbc6 100644
--- a/AuthDialog.ui
+++ b/AuthDialog.ui
@@ -114,39 +114,6 @@
      </property>
     </widget>
    </item>
-   <item row="4" column="1" colspan="3" >
-    <widget class="QCheckBox" name="cbRemember" >
-     <property name="text" >
-      <string>Remember authorization</string>
-     </property>
-     <property name="checked" >
-      <bool>true</bool>
-     </property>
-    </widget>
-   </item>
-   <item row="5" column="2" colspan="2" >
-    <widget class="QCheckBox" name="cbSessionOnly" >
-     <property name="text" >
-      <string>For this session only</string>
-     </property>
-    </widget>
-   </item>
-   <item row="5" column="1" >
-    <spacer name="horizontalSpacer" >
-     <property name="orientation" >
-      <enum>Qt::Horizontal</enum>
-     </property>
-     <property name="sizeType" >
-      <enum>QSizePolicy::Fixed</enum>
-     </property>
-     <property name="sizeHint" stdset="0" >
-      <size>
-       <width>15</width>
-       <height>10</height>
-      </size>
-     </property>
-    </spacer>
-   </item>
    <item row="1" column="1" colspan="3" >
     <widget class="KComboBox" name="userCB" />
    </item>
-- 
1.7.7.4

